openshift_tools/ansible/simplezabbix.py

```diff
@@ -21,9 +21,22 @@
 Ansible Runner and the openshift-ansible zbxapi module.
 """
 
-import ansible.runner
+from collections import namedtuple
+from ansible.parsing.dataloader import DataLoader
+from ansible.vars import VariableManager
+from ansible.inventory import Inventory
+from ansible.playbook.play import Play
+from ansible.executor.task_queue_manager import TaskQueueManager
+from ansible.plugins.callback import CallbackBase
```

Comment:
These aren't considered supported APIs, you might be better to break this out to a callback plugin and a wrapper around ansible-playbook (which is exposed as a module in Ansible 2.x)